pkg/otlp/map_provider.go

```diff
@@ -136,3 +152,9 @@ func newMapProvider(cfg PipelineConfig) (service.ConfigProvider, error) {
 	}
 	return configutils.NewConfigProviderFromMap(cfgMap), nil
 }
+
+// appendConfigMapValue appends given value to given config.Map associated to given key.
+func appendConfigMapValue(m *config.Map, key string, value []interface{}) {
+	v, _ := m.Get(key).([]interface{})
```

Comment:
I don't think we should ignore the value here, it will silently fail if we ever use this function incorrectly. 
Since any error here would be a programming error, we can just remove the second argument:

```suggestion
	v := m.Get(key).([]interface{})
```

so that we panic if we ever pass the wrong type here.